### DISCUSSION TITLE
How not to cut off zeros when rounding

### DISCUSSION THREAD

**Discussion by BlazejPalkus** [dbeaver/dbeaver#15051] [Q&A]
Hi,
I want the result to display exactly as many decimal places as I want when using the ROUND function to 4 decimal places. However, when the final numbers are zeros, they are truncated. It bothers me in data validation. Could someone please help me to solve this problem?
![image](https://user-images.githubusercontent.com/45154348/144391132-fdeb9f16-c85c-47b7-a699-dda1be810f00.png)

**Comment by Matvey16** [comment]
@mlodybanach You can disable data formatting so it will be shown as it is read from a database

![image](https://user-images.githubusercontent.com/82543000/145994591-2e013a57-e9ed-4b03-b8e2-988a32a5b223.png)
